### DISCUSSION TITLE
Port naming for module bays

### DISCUSSION THREAD

**Discussion by towdie** [netbox-community/devicetype-library#1543] [Q&A] [answered]
Hello, 
I am trying to create template for our Forcepoint firewalls which has 2 module bays. 
Port naming on this devices is that it has 8 physical ports named Interface 0-7.
When I use module bays, the port naming continues. It mean if I have line card with 2 interfaces 0-1, in the system i see port number as Interface 8-9 for the first card and Interface 10-11 for the second card.
Is there any way how I can add this to module bay template in netbox? 
Thank you for help.

**Comment by DanSheps** [comment, marked as answer]
There is no supported way to do this, I don't think.  Unfortunately, it is a replalcement variable and not a jinja template
